### PR TITLE
[SPARK-39909][SQL] Organize the check of push down information for JDBCV2Suite

### PR DESCRIPTION
**Pull request by yabola**
### What changes were proposed in this pull request?
This PR changes the check method from `check(one_large_string)` to `check(small_string1, small_string2, ...)`

### Why are the changes needed?
It can help us check the results individually and make the code more clearer. 

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
existing tests

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
Can you describe what you're optimizing in the PR description?

**Comment by yabola** [comment]
@HyukjinKwon I have updated the PR description~

**Comment by beliefer** [comment]
@yabola Please follows up the other PR. The title and description does't follows the PR specification.

File: sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala [hunk 1]
```diff
@@ -305,7 +310,8 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
       .offset(1)
     checkOffsetRemoved(df1)
     checkPushedInfo(df1,
-      "PushedFilters: [DEPT IS NOT NULL, DEPT = 1], PushedOffset: OFFSET 1,")
+      "PushedFilters: [DEPT IS NOT NULL, DEPT = 1]",
+      "PushedOffset: OFFSET 1")
```
Comment (HyukjinKwon) [inline sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala:314]: I don't think these changes are necessary. They all follow the style guides.

**Comment by cloud-fan** [comment]
Can we make the PR description more specific about the code changes? It looks to me that it changes the check method from `check(one_large_string)` to `check(small_string1, small_string2, ...)`. This looks fine as the check method eventually calls `checkKeywordsExistsInExplain`, which recommends callers put individual fragments for checking EXPLAIN result.

**Comment by yabola** [comment]
@cloud-fan Thanks for your review. I had updated the PR description。

**Review by cloud-fan — APPROVED** [review]

**Review by huaxingao — APPROVED** [review]

**Comment by huaxingao** [comment]
Merged to master. Thanks @yabola et al.

**Review by HyukjinKwon — COMMENTED** [review]
It's a nit but let's make sure keep the PR title formatted next time.

**Comment by yabola** [comment]
@HyukjinKwon @huaxingao Thanks for your review.

**Comment by huaxingao** [comment]
@yabola What's your jira id? I will assign the ticket to you.

**Comment by yabola** [comment]
@huaxingao My Jira id is miracle . I commented on the ticket~

**Pull request closed without merging**